### ISSUE TITLE
Phasing out chisel & bits in favor of little tiles

### ISSUE THREAD

**Issue by antrobot1234** [Divine-Journey-2/Divine-Journey-2#250]
**Modpack version:**
2.2.0

**Describe the request:**
I feel like the mod "Little Tiles" is a direct upgrade in terms of both features and performance compared to chisel & bits, and I feel like it would be better to make the transition into Little Tiles in the long term.

**Reasons why this should be added:**

1. while C&B stores each tile individually as a voxel, Little tiles improves upon this by combining individual voxels into larger rectangular chunks.
2. Little tiles allows structures to be saved and assigned functionality (such as doors, chairs, ladders, ect.) to give functionality to otherwise solely decoration.
3. Little tiles includes slopes, which massively increases the potential desings.
4. Little tiles inherently has forward compatability with C&B, in that any structure made in C&B can be turned into Little Tiles. This means that even users who had invested in C&B will not have their work go to waste.
5. Little tiles is actively being updated with new features and content.

**Additional context:**
While C&B is perfectly servicable for small scale building, It is simply lacking the features and optimizations of it's modern contemporary. This would especially impact servers, in that multiple players potentially using large scale buildings with C&BB would impact the performance much more significantly then Little Tiles.

**Comment by Jolain** [comment]
Agreed, Little Tiles is miles better than C&B. Would be a nice QoL improvement over C&B.

**Comment by Atricos** [comment]
It's not as easy to phase out as say Iron Backpacks.

While you can easily move items from one Backpack to the other, you can't easily rebuild your entire base if you've been using Chisels & Bits.

Also, other than performance issues, there are no major bugs with Chisels & Bits currently.

**Comment by WaitingIdly** [comment]
LittleTiles has autoconvert capability with C&B iirc

**Comment by Jolain** [comment]
> LittleTiles has autoconvert capability with C&B iirc

Yes, it does. As per the LittleTiles GitHub https://github.com/CreativeMD/LittleTiles/wiki/Mod-Compatibility
> C&B structures can be converted to LT structures and vice versa, but structures converted from LT to C&B loose any unique LT characteristics. LT tiles that contain slices or colored tiles won't be converted to C&B

**Comment by Lgmrszd** [comment]
While LT is better than C&B from my experience, some of its features might need to be disabled (door/bed tiles maybe, door (moving) tiles allow to create elevator-like structures with little to no effort), also worth checking how stable signals system is and if it gives any form of unwanted advantage.
